server/package.json

```diff
@@ -34,7 +34,8 @@
     "database:migrate": "yarn typeorm:migrate && yarn prisma:migrate",
     "database:generate": "yarn prisma:generate",
     "database:seed": "yarn prisma:seed",
-    "database:reset": "yarn database:generate && yarn database:migrate && yarn database:seed"
+    "database:reset": "yarn database:generate && yarn database:migrate && yarn database:seed",
+    "command": "node dist/src/command"
```

Comment:
There is probably a better name

Reply:
@Weiko Yes, we should put something like `twenty:migrate` or `tenant:migrate`

Reply:
@magrinj this one is to call the "command runner" app so you will do
`yarn command tenant:sync-metadata -w "2ab71a0e-7ced-495b-83df-d2197a6fdec1"` 
or
`yarn command tenant:migrate -w "2ab71a0e-7ced-495b-83df-d2197a6fdec1"` 
for example